editor/components/post-author/check.js

```diff
@@ -30,13 +29,14 @@ export default compose( [
 	withSelect( ( select ) => {
 		return {
 			postType: select( 'core/editor' ).getCurrentPostType(),
+			authors: select( 'core' ).getAuthors(),
 		};
 	} ),
 	withAPIData( ( props ) => {
-		const { postType } = props;
+		const { postType, authors } = props;
```

Comment:
Passing the props down is useless, it's done automatically in `withAPIData`

Reply:
de5b7f0a